fip-0026.md

```diff
@@ -470,120 +540,23 @@ This sets the listing fee given to the marketplace.
+|Parameter|Format|Definition|
+|---|---|---|
+|status|String|OK if successful|
+|fee_collected|Int|Amount of SUFs collected as fee|
 
 ##### Example
 ```
 {
-    "actor": "2odzomo2v4pe",
-    "marketplace": "somemarketplace"
-    "warn_days": "7"
+  "status": "OK",
+  "fee_collected": 2000000000
 }
 ```
-#### Processing
-- Request is validated per exception handling
-- Find marketplace
-- Modify warn time
-
-#### Exception handling
-|Error condition|Trigger|Type|fields:name|fields:value|Error message|
-|---|---|---|---|---|---|
-|Invalid marketplace|Supplied `marketplace` is not found in marketplace config table `mrkplconfig`|400| |"Marketplace not found"|
-|Invalid FIO domain|Supplied `fio_domain` is not listed for sale.|400| | |"FIO domain not found"|
-|Insufficient Funds|`buyer` doesn't have enough funds|400| | |"Not enough FIO"|
-|Actor not signer|The signer and the `actor` do not match |403| | |Type: invalid_signature|
-
-#### Response
-TBD
-
-##### Example
-TBD
-
-### Check Expired Listings
-
-#### New action: *chkexplistng*
-This action needs to be run periodically to check the listings to see if they are expired. If they are and the seller has a `notif_address` set, a FIO request for a small amount of FIO will be sent with a message notifying them their listing is about to expire. If there is no `notif_address` and the domain expires, the cancel listing action will be called which will remove the listing from the `domainsale` table and transfer the domain back to the seller.
-
-Anyone can call this action. It only needs to be called once a day, though 
-
-#### Processing
-- Iterate through all domains listed on `domainsale` table. 
-- If `expiration` is less than `warn_dur` days away and there is a `notif_address` on the listing
-    - If `warned_expire` is not true
-    - A notification will be sent to the `notif_address` via a FIO request about the pending expiration
-    - Flip the `warned_expire` bool to true
-- if no `notif_address` provided nothing will happen
-- if now() is greater than `expiration` then the *cxlistdomain* action is called which will transfer the domain back to the seller and delete the listing.
 
-#### Exception handling
-|Error condition|Trigger|Type|fields:name|fields:value|Error message|
-|---|---|---|---|---|---|
-|Invalid marketplace|Supplied `marketplace` is not found in marketplace config table `mrkplconfig`|400| |"Marketplace not found"|
-|Invalid FIO domain|Supplied `fio_domain` is not listed for sale.|400| | |"FIO domain not found"|
-|Insufficient Funds|`buyer` doesn't have enough funds|400| | |"Not enough FIO"|
-|Actor not signer|The signer and the `actor` do not match |403| | |Type: invalid_signature|
-
-#### Response
-TBD
-
-##### Example
-TBD
-
-### Changes to existing actions
+### Modified actions
+#### Contract: fio.token
+#### Modified Action: transfer
 
 - Added the following code to `fio.token::transfer` so that, from the fio.escrow contract, it can transfer tokens from two accounts.
```

Comment:
Why not just call transfer twice?

Reply:
I am not sure what you mean.

Reply:
You are saying that fio.token::transfer will be modified to transfer from 2 accounts at the same time. Why? You can simplyu call transfer for each transfer separately.

Reply:
Oh, my wording is unclear. 

It's not 2 transfers at the same time. The modifications of the code is to give the escrow account permission to call the transfer action instead of using the transfer by public key one. 

The current way it works it can only transfer *FROM* `SYSTEMACCOUNT` or `TREASURYACCOUNT` and then it checks to make sure any random account can only transfer *TO* `TREASURYACCOUNT` so I changed it so any account can send to any other account only when called from `SYSTEMACCOUNT`, `TREASURYACCOUNT`, or `EscrowContract` so that the buyer can send FIO to both the marketplace and the seller of the domain.

In the code, it's actually making 2 transfers, one to the marketplace account and one to the seller.